tools/RAiDER/checkArgs.py

```diff
@@ -57,7 +57,7 @@ def checkArgs(args):
             wetFilename, hydroFilename = makeDelayFileNames(
                 d,
                 args.los,
-                'nc',
+                args.raster_format,
```

Comment:
Maybe we should just call this 'format' and leave it at that. :) Can do in a different pr.

Reply:
Works for me. `output_format` just to be explicit. I'm still testing against the 2D golden datasets. So would be good to not have this hardcoded